README.md

```diff
@@ -151,7 +151,10 @@ A summary of the currently available operations in **astro**. More details are a
 ## Documentation
 
 The documentation is a work in progress, and we aim to follow the [Diátaxis](https://diataxis.fr/) system:
-* **Tutorial**: a hands-on introduction to **astro**
+* **[Python-only Tutorial](docs/QUICK_START_PYTHON.md)**: a hands-on introduction to **astro**'s python SDK.
+We'll go over basic concepts and functions for writing entire sql+python flows in your DAG file.
+* **[Python-with-sql Tutorial](docs/QUICK_START_SQL.md)**: a hands-on introduction to **astro**'s SQL rendering engine.
```

Comment:
@dimberman  My understanding is that the `render` is a feature that is part of the Python SDK , what are your thoughts?

If we agree,  perhaps we could make this more explicit by saying `SQL Python rendering engine`, what do you think? Or perhaps referring the Python SDK in this second quickstart as well?